resources/translations.js

```diff
@@ -63,6 +63,38 @@ if (typeof module !== 'undefined') {
       cn: '(?<area>\\y{Name})的封锁解除了',
       ko: '(?<area>\\y{Name})의 봉쇄가 해제되었습니다\\.',
     },
+    craftingStart: {
+      en: 'You begin synthesizing (?<recipi>\\y{Name})\\.',
```

Comment:
I've just thinking about this. Thank you!

Reply:
(；´∀｀) Sorry about this. The original regex was written by me, and I forgot the correct spell of "recipe", just wrote down the similar pronounce what I rememberred.

And another question, do you think we can match the number of count when we creating a recipe that produces multiple production?

In Japanese it would be simply "<player>は<recipe>×<count>を完成させた！"(<player> has crafted <count> <recipe>.) So I can write regex to match it in Japanese (and Chinese and Korean, too), but in France and German, that is very complecated.